utils/cli/core/core.py

```diff
@@ -202,6 +202,52 @@ def tasks_upload(self, task_id, fileformat, filename, **kwargs):
             "with annotation file {} finished".format(filename)
         log.info(logger_string)
 
+    def tasks_export(self, task_id, filename, export_verification_period=3, **kwargs):
+        """ Export and download a whole task """
+        url = self.api.tasks_id(task_id)
+        export_url = url + '?action=export'
+
+        while True:
+            response = self.session.get(export_url)
+            response.raise_for_status()
+            log.info('STATUS {}'.format(response.status_code))
+            if response.status_code == 201:
+                break
+            sleep(export_verification_period)
+
+        response = self.session.get(url + '?action=download')
+        response.raise_for_status()
+
+        with open(filename, 'wb') as fp:
+            fp.write(response.content)
+        logger_string = "Task {} has been exported sucessfully. ".format(task_id) +\
+            "to {}".format(os.path.abspath(filename))
+        log.info(logger_string)
+
+    def tasks_import(self, filename, import_verification_period=3, **kwargs):
+        """ Import a task"""
+        url = self.api.tasks + '?action=import'
+        response = self.session.post(
+                url,
+                files={'task_file': open(filename, 'rb')}
```

Comment:
Maybe add a handler if such file not exists?

Reply:
little bit improved, in this case error message will be like this, I think it's enough for user
```
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/343.zip'
```